src/PreserveModuleNamePlugin.ts

```diff
@@ -60,8 +150,10 @@ export class PreserveModuleNamePlugin {
           id = id.replace(/\\/g, "/");
           if (module.buildMeta)  // meta can be null if the module contains errors
             module.buildMeta["aurelia-id"] = id;
-          if (!this.isDll)
+          if (!this.isDll) {
+            module[preserveModuleName] = id;
```

Comment:
I think I get it, but just to be sure:
- `module` would either have `preserveModuleName` set to `id` already or `true` here.
- You force it to `id` so that it can be replaced in user source code later.

Am I correct or is there another reason for this change?

Reply:
Yeah that is correct.